src/System.Runtime.Extensions/tests/TestApp/TestApp.csproj

```diff
@@ -18,8 +18,5 @@
       <Link>Common\System\Diagnostics\CodeAnalysis\ExcludeFromCodeCoverageAssemblyAttribute.cs</Link>
     </Compile>
   </ItemGroup>
-  <ItemGroup>
-    <None Include="project.json" />
```

Comment:
A quick search of my local repo turns up a lot more instances of these project.json references.